src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -835,6 +732,142 @@ public static function stripComments(string $source)
         return $output;
     }
 
+    private function checkCacheExist(ConfigCache $cache, string $cachePath, int $errorLevel): bool
```

Comment:
the method name is confusing to me. Your implementation is not only about checking whether the cache exists. It performs the container initialization in case the cache does exist.
to me, your split actually hurts code readability, because method names are all lying about what the methods actually do.

Reply:
I've changed the name of methods and extract container from it.
help me to complete this issue because initializeContainer is too long and needs to be smaller